community/planning/circuit_purge.md

```diff
@@ -0,0 +1,124 @@
+    purged from
+  - The circuit is present in the admin store and has a `circuit_status` that
+    is not `Active`
+  - The specified circuit in the request has a `circuit_version` of at least 2
+
+Once the purge request has been validated by the admin service, the circuit and
+service state are removed from storage. This includes removing the circuit’s
+entry from the admin store. Splinter service data, specifically Scabbard’s LMDB
+files, are also removed by the purge request. All states that have been stored
+pertaining to the circuit will be removed for the requesting node. All other
+members of the circuit would still be able to access and view the circuit or
+service state. The node that has purged the circuit, however, will be unable to
+view the circuit or service data via Splinter.
+
+## Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+### `CircuitPurgeRequest`
+
+A `CircuitPurgeRequest` may be submitted to remove the state of an inactive
+circuit. An inactive circuit is a circuit with a `circuit_status` besides
+`Active`. This may follow either a `CircuitDisbandRequest` or a
+`CircuitAbandonRequest`. The `CircuitDisbandRequest` is further explained in
+the [Circuit Disband]({% link community/planning/circuit_disband.md %}) feature
+document and ultimately creates a circuit with a `Disbanded` `circuit_status`.
+The `CircuitAbandonRequest` enables a circuit to have a `circuit_status` of
+`Abandoned`. In both of these cases, the circuit networking will have been
+turned off. Once the circuit has been disbanded or abandoned, it is able to be
+purged.
+
+The `CircuitPurgeRequest` is defined in the admin protos as follows:
+
+```
+message CircuitPurgeRequest {
+    // The unique circuit id of the inactive circuit to be purged
+    string circuit_id = 1;
+}
+```
+This request is also wrapped in a `CircuitManagementPayload` before being
+submitted to the admin service. This payload would appear as follows:
+
+```
+message CircuitManagementPayload {
+    header =
+        Header {
+            action = CIRCUIT_PURGE_REQUEST;
+            requester = <Bytes of the requester’s public key>;
+            payload_sha512 = <Bytes of the hash of the payload’s action>;
+            requester_node_id = <Node ID of the requester submitting the payload>;
+        }
+    signature = <Signature of the header included in the request>;
+    circuit_purge_request = <`CircuitPurgeRequest` with the circuit ID of the circuit to be purged>;
+}
+```
+
+Once this payload is submitted to the admin service, the request is validated.
+The steps for validation are described in the previous section. If the purge
+request is invalid, the circuit and service state will remain. If the purge
+request is successfully validated, the circuit will then be removed from the
+admin store. The admin service then uses the `ServiceOrchestrator`’s
+`destroy_service` method to destroy the node’s services from the circuit being
```

Comment:
destroy has been replaced with purge_service, so this section should be updated

Reply:
Updated this section to refer to the `purge_service` method that is now being used via the `purge` command